static/skin/i18n/en.json

```diff
@@ -28,7 +28,7 @@
 	, "random-page-button-text": "Go to a randomly selected page"
 	, "searchbox-tooltip": "Search '{{BOOK_TITLE}}'"
 	, "confusion-of-tongues": "Two or more books in different languages would participate in search, which may lead to confusing results."
-	, "welcome-page-overzealous-filter": "No result. Would you like to <a href=\"?lang=\">reset filter</a>?"
+	, "welcome-page-overzealous-filter": "No result. Would you like to <a href=\"?lang=&category=&q=\">reset filter</a>?"
```

Comment:
It already handles the absence, I thought it was more consistent to give all params that time - I don't think so anymore. Will reset it.
Atlhough, with #907 changing the `?` to `#`, I think we need a overzealous-filter string for `?`, correct?

Reply:
> with #907 changing the `?` to `#`, I think we need a overzealous-filter string for `?`, correct?

Sorry, but I don't understand your question/proposal.

Reply:
https://github.com/kiwix/libkiwix/pull/907/files#diff-c70f0ae73979b109350424912c766a72199a66d69ecfa14c1afbe54dc9672ceeR31

Once this change is merged, we'll be using fragment params in js-enabled welcome page while query params in no-js page. My question was if I should be adding a new `welcome-page-overzealous-filter-query` entry with `<a href=\"?lang=\">` as link instead of `<a href=\"#lang=\">`

Reply:
Very nice :D Thanks
Added in https://github.com/kiwix/libkiwix/pull/897/commits/eac892e52330b107948082ec31f3da8babdf05d8

P.S. #907 should be merged before this PR